server/internal/app-config/config.go

```diff
@@ -147,6 +150,22 @@ func (c *Config) repoManager() error {
 			RoundStoreConfig: []interface{}{c.DbDir, logger},
 			VtxoStoreConfig:  []interface{}{c.DbDir, logger},
 		})
+	case "sqlite":
+		var database *sql.DB
+		database, err = sqlitedb.OpenDB(c.DbDir + "/sqlite.db")
+		if err != nil {
+			return err
+		}
```

Comment:
this must be moved to `db.NewService()`!

Reply:
we can pass

```
RoundStoreConfig: []interface{}{c.DbDir, logger},
VtxoStoreConfig:  []interface{}{c.DbDir, logger},
```

just like the badger db config

Reply:
done